pkg/interceptors/bitbucket/bitbucket.go

```diff
@@ -26,20 +26,27 @@ import (
 	"google.golang.org/grpc/codes"
 )
 
-var _ triggersv1.InterceptorInterface = (*Interceptor)(nil)
+var _ triggersv1.InterceptorInterface = (*BaseInterceptor)(nil)
 
-type Interceptor struct {
+type BaseInterceptor struct {
```

Comment:
So, the BaseInterceptor is the struct that implements the InterceptorInterface. Maybe we could just call it InterceptorImpl?